mbuild/port.py

```diff
@@ -83,6 +83,46 @@ def _clone(self, clone_of=None, root_container=None):
         newone.used = self.used
         return newone
 
+    def update_separation(self, separation):
+        """
+        Change the distance between a port and its anchor particle
+
+        separation : float, required
+            Distance to shift port along the orientation vector from the anchor
+            particle position. If no anchor is provided, the port will be shifted
+            from the origin.
+        """
+        if self.used:
+            warn("This port is already being used and changing its separation"
+                    " will have no effect on the distance between particles.")
+
+        if self.anchor:
+            self.translate_to(self.anchor.pos)
+        else:
+            self.translate_to((0,0,0))
```

Comment:
Here and two lines down, lets add space between the operators and commas.

`(0, 0, 0)`

Reply:
Good catch, thanks.